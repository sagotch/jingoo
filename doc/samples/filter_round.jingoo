<dl>
  <dt>1.5 | round ("floor")</dt>
  <dd>{{ 1.5 | round ("floor") }}</dd>
  <dt>1.5 | round ("ceil")</dt>
  <dd>{{ 1.5 | round("ceil") }}</dd>
</dl>
