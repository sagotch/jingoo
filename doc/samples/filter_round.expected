<dl>
  <dt>1.5 | round ("floor")</dt>
  <dd>1.</dd>
  <dt>1.5 | round ("ceil")</dt>
  <dd>2.</dd>
</dl>
